### DISCUSSION TITLE
How do I display seconds on the clock?

### DISCUSSION THREAD

**Discussion by ProfBrianChen** [valinet/ExplorerPatcher#1392] [General]
I saw this feature in the all-features page, but I cannot figure out how to turn it on.  Can anyone explain?

**Comment by pyrates999** [comment]
Open up explorer patcher properties.  You can do so by right clicking on the task bar and selecting properties.

Then go to system tray and look for "show seconds in the clock".  Click beside it so it is a tick mark instead of an X.

Then click on restart file explorer at the bottom left.

**Reply by ProfBrianChenLehigh** [reply]
Thanks for your precise response. I'm on 22621.1555.55.1. The System tray item on the left provides a list with six items: "Skin taskbar and try popup..", "Center try icon pop-up menus", "flyout behavior for ..." "Show touch keyboard button", "control center button", and "show desktop button".

Regrettably, the "Show seconds in the clock" item is not there.  Can you explain what's wrong?

**Reply by feiudfyg** [reply]
hey i found a tool that can show seconds of the clock.
https://github.com/Chuyu-Team/Dism-Multi-language
if it doesnt work, reply me

**Reply by feiudfyg** [reply]
btw show seconds only works when you are using win10 style taskbar.

**Comment by pyrates999** [comment]
What version of windows are you running?  You can find out by running "winver" from run.
What version of explorer patcher are you running?  You can find out by right clicking on the task bar and selecting properties.  Then click on about in the left hand side.  Should be 22621.1555.55.1 as of May 1.

I suspect the seconds option for the clock is not supported in windows 11.

**Comment by vippium** [comment]
If you are on the Latest build (22621.1635) and want to enable this on Win11 Taskbar, then, simply go to Registry Editor and navigate here : **HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\Advanced**, Create a DWORD named **ShowSecondsInSystemClock** and set the value to **1**

But currently, there is no direct way to Enable or disable this. You have to go to registry editor to disable it again.

**Close Issue if It is Resolved**

**Reply by ProfBrianChenLehigh** [reply]
Thanks.  It appears that I am on a similar build but slightly earlier than that.  I'll try updating.  I actually already had the registry key set to 1, surprisingly

**Reply by pyrates999** [reply]
22621.1635 I think is a pre-release.  So I don't think this will work since the release is 22621.1555.55.1.
